packages/dm-core-plugins/src/form/templates/shared/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { TUiAttributeObject } from '../../types'
+
+export const ExpandViewConfig = (uiAttribute?: TUiAttributeObject) => {
+  return (
+    uiAttribute?.expandViewConfig ?? {
+      type: 'ReferenceViewConfig',
+      recipe: uiAttribute?.uiRecipe,
+    }
+  )
+}
+
+export const OpenViewConfig = (
+  uiAttribute?: TUiAttributeObject,
+  namePath?: string
+) => {
+  return (
+    uiAttribute?.openViewConfig ?? {
+      type: 'ReferenceViewConfig',
+      scope: namePath,
+      recipe: uiAttribute?.uiRecipe,
+    }
+  )
+}
+
+export const InferCanOpenOrExpand = (
+  objectIsNotEmpty: boolean,
+  config: any,
```

Comment:
Do we perhaps have a proper type for what the config can be? If not, it would be something like `Record<string, unknown>`

Reply:
Hmm, then we can't use the dot notation like we do. 
Ofc, we can change it to ["functionality"] and use the Record syntax. 
What do you think is the best approach? 


We could do partial typing though, sprecifying that we know that it at least has these props. 

We have a lot of configs that can be many things in the application. I don't have a good suggestion yet of what to do about them.  

<img width="621" alt="image" src="https://github.com/equinor/dm-core-packages/assets/37016135/6f1f9b94-babe-433b-bad6-c9be02779eec">